skbio/io/fastq.py

```diff
@@ -449,9 +449,9 @@ def _alignment_to_fastq(obj, fh, variant=None, phred_offset=None,
 
 def _line_generator(fh):
     for line in fh:
-        line = line.rstrip('\n')
+        line = line.strip()
```

Comment:
Can you update the fastq docs to match this new behavior? The current docs state that lines aren't stripped.